crates/rpc/src/method/simulate_transactions.rs

```diff
@@ -1436,8 +1602,12 @@ pub(crate) mod tests {
         let result = simulate_transactions(context, input).await.unwrap();
 
         pretty_assertions_sorted::assert_eq!(
-            result,
-            SimulateTransactionOutput(vec![
+            result
+                .serialize(Serializer {
+                    version: RpcVersion::V06
```

Comment:
Same question: I think these serializers reflect the 0.7 JSON-RPC spec, so we should probably be using `V07` here? (I understand that it doesn't really make a difference right now.)